module/VuFind/tests/unit-tests/src/VuFindTest/SMS/ClickatellTest.php

```diff
@@ -76,6 +76,41 @@ public function testCarriers()
         $this->assertEquals($expected, $obj->getCarriers());
     }
 
+    /**
+     * Test unknown exception message error
+     *
+     * @return void
+     */
+    public function testUnknownException()
+    {
+        $client = $this->getMockClient($this->getMockClient(), []);
```

Comment:
I think something is wrong here -- not sure why `getMockClient` is getting passed in to `getMockClient`.